docs/data/date-pickers/lifecycle/lifecycle.md

```diff
@@ -21,7 +21,7 @@ The field components have an internal state controlling the visible value.
 It will only call the `onChange` callback when:
 
 - the user completes all sections of a field. The value reflects the field.
-  - the value could be `Invalid Date` in case of an unparsable date (i.e., if the [year is less than "100"](https://github.com/iamkun/dayjs/issues/1237) in case of `dayjs`).
+  - the value will be `Invalid Date` in case of an unparsable date (for example, if the [year is less than "100"](https://github.com/iamkun/dayjs/issues/1237) in case of `dayjs`).
```

Comment:
Vale...

```suggestion
  - when the date is not parseable, the `onChange` prop receives an `Invalid Date` (for example, if the [year is less than "100"](https://github.com/iamkun/dayjs/issues/1237) in case of `dayjs`).
```

If you prefer :laughing: